src/http/mod.rs

```diff
@@ -15,12 +20,15 @@ use crate::document::Document;
 use crate::metrics;
 use crate::rpc::error::Errors;
 
+use hyper::client::connect::dns::GaiResolver;
+use hyper::Body;
+
 use self::filters::UriFilter;
 
 pub mod filters;
 
 pub struct HttpClient {
-    client: Client<HttpsConnector<HttpConnector>>,
+    client: Client<TimeoutConnector<HttpsConnector<HttpConnector<GaiResolver>>>, Body>,
```

Comment:
Would it be possible to remove the `GaiResolver`generic here? The previous statement didn't have that.

Reply:
I tried to keep it simple but using Hyper-timeout forced me to use GaiResolver, else it wasn't working with with any other type of client